src/reactions/persist.js

```diff
@@ -0,0 +1,22 @@
+import { reaction } from 'mobx';
+import { findByProperty } from 'util/common';
+import { persistKey } from 'util/persistors';
+import { FilterStates, PrefCookieKeys } from 'stores/prefs-store';
+
+export default function ({ prefs_store }) {
+  const keys = ['statusFilter', 'sortCriteria', 'sortDirection', 'compact']
+  keys.forEach(k => {
+    reaction(
```

Comment:
Here we're creating a different reaction for every key. We can improve that by creating a single reaction that tracks required observables as follows:

```javascript
const prefKeys = ['statusFilter', 'sortCriteria', 'sortDirection', 'compact'];

reaction(
  prefKeys.map((prefKey) => ({[prefKey]: prefs_store[prefKey]})),
  prefPair => {
    prefPair.forEach(serializePref);
  }
)
```